Sources/JWT/Signers/RSASigner.swift

```diff
@@ -26,67 +26,117 @@ extension HashMethod {
     }
 }
 
-public struct RS256: RSASigner {
-    public let key: Bytes
+public typealias CRSAKey = UnsafeMutablePointer<RSA>
```

Comment:
I'm not sure if RSASigner in general should be public. I think it would be cleaner to just expose the supported schemes (RS256, etc.) directly, not the fact that they are implemented using the same base.